app/src/views/private/components/module-bar.vue

```diff
@@ -9,8 +9,8 @@
 				v-tooltip.right="modulePart.name"
 				icon
 				x-large
-				:to="modulePart.to"
-				:href="modulePart.url"
+				:to="modulePart.to || (modulePart.url && modulePart.url.startsWith('/') ? modulePart.url : undefined)"
```

Comment:
Internal URLs could be tricky.

I've just tested your changes:

https://user-images.githubusercontent.com/14039341/133894496-65d1e602-9db6-4216-b745-7185d518b5bd.mp4

So I had put `/admin/collections/category` on link and that led me to `/admin/admin/collections/category`.

I think you could check if it starts with `/admin` and remove it, what do you think?

Reply:
Thank you @joselcvarela for helping to test the PR.

If it is a relative (internal as you call it) URL, users can just use `/collections/category`.
I personally think that doing a check for `/admin` and removing it would result in a messier codebase.
All the routes from modules are without the `/admin` as they are all definitely under the admin folder.

In addition, I would believe that for most people upon seeing their link not working and the URL having `/admin/admin`, they would remove it (as they added in the first place).

On a possibly relevant note, there is a question in the Discord SaaS channel asking about changing the `/admin` folder to just `/`. I am not sure if that is possible or not, but should it be possible, I suppose we should not be hardcoding the detection of `/admin`.

Would a note stating that relative URLs should exclude `/admin` suffice in your opinion?

Reply:
From a developer perspective, you are right, we should avoid hardcoded strings
But from an end user perspective, I also expect that a relative link is always relative to the domain, and not to a nested path.

So I don't know if a note is a little bit "hacky" or if we should create an interface for that like this:
![image](https://user-images.githubusercontent.com/14039341/133896246-c0046324-02e7-46d8-aa67-0fc683f07bf0.png)

What are your thoughts @rijkvanzanten @benhaynes ?

Reply:
> But from an end user perspective, I also expect that a relative link is always relative to the domain, and not to a nested path.

Completely agree with this.

Refer the below screenshot from your video, the links for the modules do not include `/admin`.
<img width="629" alt="image" src="https://user-images.githubusercontent.com/26413686/133896330-79e10219-daf8-44b1-8260-ffda3392612e.png">

Perhaps it is just due to the term relative URLs. 
Would it still be an issue if it is termed as internal URLs.

Your interface idea looks good but it may be limiting as there are `mailto:`, `tel:`, or other deep linking custom schemes that work on both desktop and mobile.

Reply:
I like that approach, @joselcvarela !! And a good thing to keep in mind, while it's not on our immediate roadmap, would be if `/admin` ever changed or became configurable. In that case we want to avoid that being in these module strings. I love the idea of the drop-down prefix interface! Seems like a nice option to have regardless. 🙌